packages/react-table/src/components/Table/utils/decorators/headerCol.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { IExtra, IFormatterValueType, ITransform } from '../../Table';
 
 export const headerCol = (id = 'simple-node') => {
+  // tslint:disable-next-line:no-shadowed-variable
```

Comment:
I don't think this is needed since `headerCol` and `headerColObj` are two different variable names.